managed-apps-vision/README.md

```diff
@@ -0,0 +1,30 @@
+# Managed Apps Vision
+
+This RFC describes a general vision for improving the user experience of Managed Apps.
+
+With a previous iteration of the structure of Giant Swarm, one team was responsible for all of the managed apps. As the number of apps that Giant Swarm managed increased, it became more difficult to own these apps.
+
+Giant Swarm has updated its structure so that ownership of apps is spread across multiple teams - instead of one team owning all apps, each team owns a number of apps. Each team is now responsible for an area of solutions, such as monitoring, and are then responsible for apps that help in that area, such as Prometheus. This ownership also includes concepts such as trainings or sales enablement. We've been referring to this concept as 'holistic ownership'.
+
+With this change, more teams and more people are working on apps, overall. This means we can spend more time on each app, and provide a better user experience overall - we can start working more _deeply_.
+
+As some examples of what working more deeply on apps means:
+- Working upstream to fix specific customer issues, or provide feedback to maintainers on issues with upgrade paths.
+- Building more application specific monitoring and alerting, so Giant Swarm is aware of application level issues before customers.
+- Improving support for automatic scaling, so apps don't OOM on larger clusters, or scale on custom metrics.
+- Building dashboards (or reusing upstream dashboards) and making them available to customers.
+- Working with the incident and postmortem workflows to ensure that issues with applications are learnt from and improved upon.
+
+These are just examples - it is up to each team to identify how to provide a better experience with the apps they manage. Overall, we can should be aiming to improve our operations of managed apps, and reduce the amount of work our users need to do.
```

Comment:
```suggestion
These are just examples - it is up to each team to identify how to provide a better experience with the apps they manage. Overall, we are aiming to improve our operations of managed apps, and reduce the amount of work our users need to do.
```

Reply:
Went with 'should be aiming', that makes more sense to me with what i wanna get across